clustered/server/src/main/java/org/ehcache/clustered/server/store/WriteBehindLockManagerImpl.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.server.store;
+
+import org.terracotta.entity.ClientDescriptor;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+public class WriteBehindLockManagerImpl extends LockManagerImpl {
+  @Override
+  public void sweepLocksForClient(ClientDescriptor client, Consumer<List<Long>> removeHeldKeys) {
+  }
```

Comment:
The implementation is empty. Why do you even need this? sweeping of locks for the client leaving will happen irrespective of whether is write-behind or just a simple clustered loader-writer.

Reply:
Need to push a change that removes locks held by the disconnected client. The difference for WriteBehind cache is that cache entries won't be removed.